host/src/prover/execution.rs

```diff
@@ -20,12 +20,18 @@ pub async fn execute<D: Prover>(
     req: &ProofRequest<D::ProofParam>,
 ) -> Result<D::ProofResponse> {
     println!("- {:?}", req);
-    // 1. load input data into cache path
+    // Generate the witness
     let start = Instant::now();
     let input = prepare_input(ctx, req).await?;
-    let elapsed = Instant::now().duration_since(start).as_millis() as i64;
-    observe_input(elapsed);
-    // 2. pre-build the block
+    let time_elapsed = Instant::now().duration_since(start);
+    println!(
+        "=> Witness generated in {}.{} seconds",
+        time_elapsed.as_secs(),
+        time_elapsed.subsec_millis()
+    );
+    observe_input(time_elapsed.as_millis() as i64);
```

Comment:
shall we remove this? useless :(
and perhaps the whole https://github.com/taikoxyz/raiko/blob/d378dd491957b59f943091262836abc0fc709285/host/src/main.rs#L20

Reply:
Yeah kept it in for now because I am not sure of the purpose, removing it works for me.